{"uuid": "1e8b704f-8001-4a99-a8ae-4be4560cd6a6", "children": ["98cf33ed-474d-4590-86b8-b59b19a08fee"], "befores": [{"name": "capture_screenshot_on_failure", "status": "passed", "start": 1742900719935, "stop": 1742900719936}], "afters": [{"name": "capture_screenshot_on_failure::0", "status": "broken", "statusDetails": {"message": "AttributeError: 'Function' object has no attribute 'rep_call'\n", "trace": "  File \"C:\\Users\\rutwik\\miniconda3\\envs\\automation\\lib\\site-packages\\allure_commons\\_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"C:\\Users\\rutwik\\miniconda3\\envs\\automation\\lib\\site-packages\\_pytest\\fixtures.py\", line 907, in _teardown_yield_fixture\n    next(it)\n  File \"C:\\Users\\rutwik\\Automation\\Automation-logic\\conftest.py\", line 102, in capture_screenshot_on_failure\n    if request.node.rep_call.failed:\n"}, "start": 1742900747838, "stop": 1742900747872}], "start": 1742900719935, "stop": 1742900747873}